src/base64.h

```diff
@@ -22,7 +22,7 @@ namespace base64 {
 
 std::string Encode(const std::string &source);
 
-std::string Decode(const std::string &source);
+//std::string Decode(const std::string &source);
```

Comment:
Why did you decide to comment this out?

Reply:
The definition is commented out, so I thought I would reflect that in the header file.  It seemed wrong to have a declaration with no definition anywhere.

I don't feel tied to this - happy to change it back.